src/app/[locale]/dataset-list.cy.tsx

```diff
@@ -1,36 +1,72 @@
-import DatasetList from './dataset-list';
+import DatasetList, {Props} from './dataset-list';
+import {NextIntlClientProvider} from 'next-intl';
+import messages from '@/messages/en.json';
 
-describe('<DatasetList />', () => {
-  context('List rendering', () => {
-    it('Shows 1 dataset card if one dataset is returned in the search result', () => {
-      cy.fixture('search-results/one-dataset.json').then(searchResult => {
-        cy.intercept({method: 'GET', url: '/api/datasets*'}, searchResult);
-        cy.mount(
-          <DatasetList initialSearchResult={searchResult} locale="en" />
-        );
-        cy.getBySel('dataset-card').its('length').should('eq', 1);
-      });
+function DatasetListWithTranslation(props: Props) {
+  return (
+    <NextIntlClientProvider locale="en" messages={messages}>
+      <DatasetList {...props} />
+    </NextIntlClientProvider>
+  );
+}
+
+describe('List rendering', () => {
+  it('Shows 1 dataset card if one dataset is returned in the search result', () => {
```

Comment:
Minor: I'm used to starting the `it()` lowercased, so that the entire test name reads like a sentence:

'List rendering shows 1 dataset card if one dataset is returned in the search result'

This is also used by Next (e.g. [this one](https://github.com/vercel/next.js/blob/canary/test/e2e/app-dir/app-basepath/index.test.ts#L16) or [this one](https://github.com/vercel/next.js/blob/canary/examples/with-cypress/cypress/e2e/app.cy.ts#L8)).

Would it be possible to follow this approach? Or would you recommend against it?